nengo_gui/config.py

```diff
@@ -4,21 +4,33 @@
 
 import nengo_gui.components
 
+def make_param(name, default):
+    try:
+        # the most recent way of making Parameter objects
+        p = nengo.params.Parameter(name=name, default=default)
+    except:
```

Comment:
Good catch...  I've started #696 to collect this and any other changes people suggest